babel.config.js

```diff
@@ -4,24 +4,13 @@ module.exports = function (api) {
     parserOpts: {
       strictMode: true,
     },
+    targets: {
+      browsers: ['chrome >= 66', 'firefox >= 68'],
+    },
```

Comment:
@Gudahtt pinging you on this, just wanted to alert you of this change as it will likely impact #12847. This was changed in babel to support making the targets available to *all* plugins and presets instead of just preset-env.